Scripts/Helpers/VRSmoothCamera.cs

```diff
@@ -79,6 +79,13 @@ void Start()
             m_Rotation = transform.localRotation;
         }
 
+        void OnEnable()
+        {
+            // Prevent camera from performing a very fast (shortest path) rotational correction when being enabled
+            if (m_VRCamera)
+                m_Rotation = m_VRCamera.transform.localRotation;
```

Comment:
Why not Start? And when would VRCamera be null?

Reply:
@mtschoen-unity  It's in OnEnable because the VRCam's current localRotation needs to be fetched and set as the new starting position to (smooth)lerp against, each time the presentation cam is enabled.  If this isn't performed, the camera rotation will perform quick "jumps" from the previously cached local rotation value.

Reply:
m_VRCamera is null in this case when starting EXR